cmd/tealdbg/localLedger.go

```diff
@@ -130,6 +161,26 @@ func makeAppLedger(
 	return ledger, states, err
 }
 
+func getBalanceFromIndexer(indexerURL string, account basics.Address, round int) (basics.AccountData, error){
+	queryString := fmt.Sprintf("%s/v2/accounts/%s?round=%v", indexerURL, account, round) // path.Join(indexerURL, "v2", "accounts", fmt.Sprintf("%s?round=%v", acc, round))
+	fmt.Println(queryString)
+	resp, err := http.Get(queryString)
+	if err != nil {
+		return basics.AccountData{}, err
+	}
+	var accountResp AccountIndexerResponse
```

Comment:
you need to check that the response code is 200. if it's not, there is no point in going forward.